### PR TITLE
add new test file to cover the routes

### PR DESCRIPTION
**Pull request by danielevalverde**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context, type of change (bug fix, performance, feature, etc.) and things to pay attention to during review.

References: TICKET-ID-1, TICKET-ID-2, …, TICKET-ID-N

## How to test?

Please describe how to test the changes (manually and/or automatically).

## Checklist

- [ ] I have performed a self-review of my code and ensured that it is runnable. I have also followed [Meedan's internal coding guidelines](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1309605889/Coding+guidelines).

### CODE REVIEW

File: test/spec/routes_spec.rb [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    user = data[:user2]
+    team = data[:team]
+
+    # log in with user 2
+    @driver.navigate.to "#{api_path}session?email=#{user.email}"
+    sleep 2
+
+    @driver.navigate.to "#{@config['self_url']}/#{team.slug}/feed/#{data[:feed_invitation].feed['id']}/invitation"
+    wait_for_selector('.feed-invitation-container-card')
+    expect(@driver.page_source.include?('This page does not exist or you do not have authorized access.')).to be(false)
```
Comment (amoedoamorim) [inline test/spec/routes_spec.rb:216]: Hi @danielevalverde! I noticed you're testing that the "not found" page isn't displayed. Please also add to another assertion to each test for negative "crash" page content as well: "An unexpected error happened"

**Review by amoedoamorim — CHANGES_REQUESTED** [review]
Please also ensure the "crash" page contents are not displayed for each route

**Review by amoedoamorim — APPROVED** [review]
:lgtm:
